.travis.yml

```diff
@@ -280,3 +280,8 @@ matrix:
         - echo 'Checking that there is no GPL licence text in code'
         - ! git grep -q --ignore-case "gnu general public";
         - ! git grep -q --ignore-case "gnu library general public";
+    - env:
+        - NAME=psa-autogen
+      script:
+        - python tools/spm/generate_partition_code.py
```

Comment:
What this supposed to be doing ? Checking unstaged files (means the files have changed?) A comment here might help (also this will need docs PR - there're travi sjobs described)

Reply:
Added a comment
Basically it runs the code generation tool and checks if changes are required